shell/browser/bluetooth/electron_bluetooth_delegate.cc

```diff
@@ -23,6 +27,28 @@ using content::RenderFrameHost;
 using content::WebContents;
 using device::BluetoothUUID;
 
+namespace gin {
+
+template <>
+struct Converter<content::BluetoothDelegate::PairingKind> {
+  static v8::Local<v8::Value> ToV8(
+      v8::Isolate* isolate,
+      content::BluetoothDelegate::PairingKind paring_kind) {
+    switch (paring_kind) {
+      case content::BluetoothDelegate::PairingKind::kConfirmOnly:
+        return StringToV8(isolate, "confirm");
+      case content::BluetoothDelegate::PairingKind::kConfirmPinMatch:
+        return StringToV8(isolate, "confirmPin");
+      case content::BluetoothDelegate::PairingKind::kProvidePin:
+        return StringToV8(isolate, "providePin");
+      default:
+        return StringToV8(isolate, "unknown");
```

Comment:
question: Is there a reason we're limiting the enum to these three values? I see that `content::BluetoothDelegate::PairingKind` has two more types that aren't exposed here.

```cpp
  enum class PairingKind {
    kConfirmOnly,
    kConfirmPinMatch,
    kDisplayPin,
    kProvidePasswordCredential,
    kProvidePin
  };
```

https://source.chromium.org/chromium/chromium/src/+/main:content/public/browser/bluetooth_delegate.h;l=59-65?q=content::BluetoothDelegate::PairingKind&ss=chromium

Reply:
@erickzhao Chromium doesn't use those other values.  When/if they start using them we can update our code, but I thought for now it would be confusing to have unused values.

Reply:
Thanks! That explains a lot.